frontend2/schema.yml

```diff
@@ -2486,9 +2486,6 @@ components:
       properties:
         profile:
           $ref: '#/components/schemas/TeamProfilePrivateRequest'
-        episode:
-          type: string
-          minLength: 1
     PatchedUserPrivateRequest:
```

Comment:
This change came from rebasing onto main. There was a bugfix that disabled the ability to change an team's episode.